api_tests/lib/ledger_runner.ts

```diff
@@ -171,15 +171,8 @@ async function startBitcoinContainer(): Promise<StartedTestContainer> {
 }
 
 async function startEthereumContainer(): Promise<StartedTestContainer> {
-    return new GenericContainer("parity/parity", "v2.5.0")
-        .withCmd([
-            "--config=dev",
```

Comment:
Reposting my comment here since it got ignored in the other thread:

We do we no longer pass all the other configuration options?